basics/getting_started.md

```diff
@@ -35,10 +39,12 @@ languages in Bazel right now.
 
       For faster iteration times (but larger binaries), use `//src:bazel-dev.exe` instead.
 
-*  This will produce a working Bazel binary in `bazel-bin/src/bazel` (or `bazel-bin/src/bazel.exe` on Windows).
+*  This produces a working Bazel binary in `bazel-bin/src/bazel` (or `bazel-bin/src/bazel.exe` on Windows).
```

Comment:
This doesn't appear to be a step, but is the end of installing. Take it out of the list.

Reply:
Done.